pkg/targets/adapter/splunktarget/adapter_test.go

```diff
@@ -58,36 +59,43 @@ func (c *mockedSplunkClient) LogEvent(in *splunk.Event) error {
 func TestReceive(t *testing.T) {
 	testCases := map[string]struct {
 		client       *mockedSplunkClient
-		expectResult cloudevents.Result
+		expectResult *errorEvent
```

Comment:
`expectEvent` maybe?
Also shouldn't we still check the value of `cloudevents.Result`?

Reply:
got it!